epoch_coordinator/README.md

```diff
@@ -2,9 +2,6 @@
 
 This component has some logic but important function of it is "wrapping" etcd.
 
-Since it's a "wrapper" around etcd it's hard to test it without etcd like we do for tests/integration (inside one process). That's why we have here `./docker/etcd` folder that contains all the necessary files to run etcd cluster in docker compose.
+You don't need to run containers yourself, you just need to have docker installed and running on your host.
```

Comment:
👍 

Finally got to the real keyboard after reading this code from the phone.

Quick question: is this comment even needed? I.e., if Docker is not running, what will the error message look like?

If it's a reasonable error message, I'd say an even shorter "you need Docker installed to run these tests", without any "just" or "don't need to" will be good enough.